spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/config/it/GcpConfigIntegrationTests.java

```diff
@@ -27,7 +27,8 @@
 import org.springframework.context.ConfigurableApplicationContext;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assumptions.assumeThat;
```

Comment:
May be it's time to ask Spring Cloud team to include this `Assumptions` for static imports as well?

Reply:
Ok, I have created https://github.com/spring-cloud/spring-cloud-build/pull/85 requesting to add Awaitility and Assumptions to the spring-cloud-build checkstyle.

For now, I'll put this PR on hold and wait to see if the changes are accepted.

Reply:
spring-cloud/spring-cloud-build#85 was just merged; I am going to close this PR and redo PR to reflect those changes.